src/components/PopupWithForm.js

```diff
@@ -4,6 +4,20 @@ export default class PopupWithForm extends Popup {
     super({ popupSelector });
     this._handleFormSubmit = handleFormSubmit;
     this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._popupDeleteForm =
+      this._popupElement.querySelector("#delete-modal-form");
+    this._popupButton = this._popupElement.querySelector("#delete-sure-button");
+
+    this._pictureChangeButton = this._popupElement.querySelector(
+      "#profile-change-button"
+    );
+    this._pictureChangeModal = this._popupElement.querySelector(
+      "#picture-modal-form"
+    );
+    this._pictureChangeSubmit = this._popupElement.querySelector(
+      "#picture-save-button"
+    );
```

Comment:
Please, delete all hardcode from **universal** `PopupWithForm`


There should **not** be any code in universal classes that doesn’t suit every popup.